azure-pipelines/build.yaml

```diff
@@ -49,7 +49,7 @@ steps:
   # hence don't set PYTHONPATH
   - bash: |
       source activate InnerEye
-      pytest ./Tests/ -m "not (gpu or azureml or after_training_single_run or after_training_ensemble_run or inference or after_training_2node or after_training_glaucoma_cv_run)" --doctest-modules --junitxml=junit/test-results.xml --cov=. --cov-config=.coveragerc --cov-report=xml -n 2 --dist=loadscope --verbose
+      pytest ./Tests/ -m "not (gpu or azureml or after_training_single_run or after_training_ensemble_run or inference or after_training_2node or after_training_glaucoma_cv_run or after_training_hello_container)" --doctest-modules --junitxml=junit/test-results.xml --cov=. --cov-config=.coveragerc --cov-report=xml -n 2 --dist=loadscope --verbose
```

Comment:
Maybe this could be done better by https://pypi.org/project/pytest-unmarked/ ? See this SO question: https://stackoverflow.com/questions/39846230/how-to-run-only-unmarked-tests-in-pytest

Reply:
This is a bit complicated - there does not seem to be an easy way to check for custom markers only and ignore markers such as "skipif" and "parametrize".